lib/upgrade_html.js

```diff
@@ -589,4 +595,87 @@ function insertHtmlCommentBefore($, elem, commentLines) {
   $(elem).before(commentText);
 }
 
+function upgradeGlobalCss($) {
+  var cssRules = {};
+  $(upgradeCss.needsUpgradeQuerySelector).each(function(_ignored, elemNeedsUpgrade) {
```

Comment:
I was going to say that you should use `_` for ignored arguments, but then realized you're using underscore, and then realized how annoying it is that they stole the `_` convention for unused args...